<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qimageuploader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../qimageuploader.qrc">
    <normaloff>:/res/icon_main.ico</normaloff>:/res/icon_main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_files"/>
   <addaction name="actionSettings"/>
   <addaction name="actionScreenshot"/>
   <addaction name="actionGrab_frames"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_files">
   <property name="icon">
    <iconset resource="../../qimageuploader.qrc">
     <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
   </property>
   <property name="text">
    <string>Share files</string>
   </property>
  </action>
  <action name="actionAdd_Images">
   <property name="icon">
    <iconset resource="../../qimageuploader.qrc">
     <normaloff>:/res/images.ico</normaloff>:/res/images.ico</iconset>
   </property>
   <property name="text">
    <string>Share files</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../qimageuploader.qrc">
     <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionGrab_frames">
   <property name="icon">
    <iconset resource="../../qimageuploader.qrc">
     <normaloff>:/res/grabmovie.ico</normaloff>:/res/grabmovie.ico</iconset>
   </property>
   <property name="text">
    <string>Import video</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../qimageuploader.qrc"/>
 </resources>
 <connections/>
</ui>
